src/Prowlarr.Api.V1/ProviderResource.cs

```diff
@@ -39,8 +39,8 @@ public virtual TProviderResource ToResource(TProviderDefinition definition)
                 Fields = SchemaBuilder.ToSchema(definition.Settings),
 
                 //prowlarr/supported#{0} are custom build redirect pages; if passing a new var, create a new redirect
-                InfoLink = string.Format("https://wikijs.servarr.com/prowlarr/supported#{0}",
-                    definition.Implementation.ToLower())
+                InfoLink = string.Format("https://wikijs.servarr.com/prowlarr/supported-indexers#{0}",
```

Comment:
Does this link actually work (e.g. the #torrentgalaxy) as they aren't headers...

Reply:
yeah still need to figure that out on the wiki since we're using a table... also need to make this conditional to only apply to indexers; not all implementations